src/core/ui/components/drop-down/inner.tsx

```diff
@@ -166,32 +168,60 @@ export const Inner = ({ context }: Props) => {
                 !user.wallet && styles.action,
                 isManagedWallet && styles.managedWallet
               )}
+              data-tip="Wallet managed by SlashAuth"
+              data-for="managed-wallet-tooltip"
             >
               {user.wallet
                 ? shortenEthAddress(user.wallet)
                 : '+ Connect Wallet'}
             </span>
-            {isManagedWallet && (
-              <div
-                className="ml-1"
-                data-tip="Wallet managed by SlashAuth"
-                data-for="managed-wallet-tooltip"
-                data-iscapture="true"
+            <div data-tip="Check on Etherscan" data-for="etherscan-tooltip">
```

Comment:
If I'm reading this correctly, this element will show up even if the user does not have a wallet